### ISSUE TITLE
oobe error

### ISSUE THREAD

**Issue by truefriend-cz** [DrEmpiricism/Optimize-Offline#54]
After instalation ended error:
https://pbs.twimg.com/media/DolWzoyW4AA2q3j.jpg
OOBEKEYBOARD

**Comment by truefriend-cz** [comment]
After click "Skip" show next error "OOBEZOP".

**Comment by DrEmpiricism** [comment]
I have never received this error with Optimize-Online, and have only received similar errors when running fairly advanced setup scripts using SetupComplete.cmd

**Comment by truefriend-cz** [comment]
Yes. No. It is have problem is official ISO downloaded from Microsoft servers. Maybe drivers conflict from (1709)-1803 builds.
I tested. I tried to install the official build on the problematic PC and it also showed the errors mentioned.
Problem is not from your script.

**Comment by DrEmpiricism** [comment]
Where are you getting this ISO?  I can probably audit the errors from my server as it installs via SCCM deployment and see where it's getting hug up.
